CRM/Civicase/Hook/PermissionCheck/CaseCategory.php

```diff
@@ -123,8 +123,9 @@ private function modifyPermissionCheckForCategory($permission, &$granted, $caseC
    */
   private function shouldRun($permission, $granted) {
     $defaultCasePermissions = array_column($this->caseCategoryPermission->get(), 'name');
+    $isNotBasicCase = $permission != 'basic case information';
 
-    return in_array($permission, $defaultCasePermissions) && !$granted;
+    return in_array($permission, $defaultCasePermissions) && !$granted && $isNotBasicCase;
```

Comment:
can we ask @tunbola to review this bit? I'm not familiar with this class.

Reply:
This was added by him only :)